backend.tf

```diff
@@ -21,8 +16,8 @@ resource "aws_s3_bucket" "terraform-state-storage-s3" {
 resource "aws_dynamodb_table" "dynamodb-terraform-state-lock" {
   name = "wca-terraform-state-lock-dynamo"
   hash_key = "LockID"
-  read_capacity = 20
-  write_capacity = 20
+  read_capacity = 5
```

Comment:
Could this lead to downtime?

Reply:
this dynamodb table is just there so two people don't update the terraform template at once (I honestly think it's quite unnecessary anyway). But it can't lead to downtime for wca.link has it's unrelated